src/Options/Options.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import { browser } from 'webextension-polyfill-ts';
+import { LIKER_LAND_URL } from '../constant';
 
-const Options = () => {
-    return <div>Hello World</div>;
-};
+
+const civicLikerURL = `${LIKER_LAND_URL}/civic?utm_soure=extension`;
+const logoutURL = `${LIKER_LAND_URL}/logout`;
+
+const background = browser.extension.getBackgroundPage();
+
+class Options extends React.Component {
```

Comment:
There is a convention that the component name should consist of two words or more, to distinct from the HTML element
for this case, I suggest renaming to `OptionsScreen`

Reply:
how about `OptionsPage` which is how it is called in chrome/mdn documentation?